TradeItIosTicketSDK2/TradeItTradingTicketViewController.swift

```diff
@@ -81,12 +89,20 @@ class TradeItTradingTicketViewController: TradeItViewController, UITableViewData
             )
             return
         }
-
+        
```

Comment:
@falnatsheh what's going on with your Xcode whitespace trimmer!

Reply:
So this is inside a function to separate logical group.

Reply:
Yeah but these are just blank space characters